packages/esm-appointments-app/src/helpers/functions.ts

```diff
@@ -53,7 +53,7 @@ export const getTodaysAppointment = (appointment: Appointment, t?: Function) =>
     patientUuid: appointment.patient.uuid,
     dateTime: formatAMPM(parseDate(appointment.startDateTime)),
     serviceType: appointment.service ? appointment.service.name : '--',
-    serviceColor: appointment.service ? appointment.service.color : '--',
+    serviceColor: appointment.service?.color ? appointment.service.color : '#006400',
```

Comment:
It would be best to not use magic colors. Do you think it makes sense to format this to have a default class of which uses a styleguide defined color and then have an inline style if appointment.service.color is present?